src/Orleans.Runtime/MembershipService/MembershipOracle.cs

```diff
@@ -142,7 +144,7 @@ public async Task BecomeActive()
                 MembershipTableData table = await membershipTableProvider.ReadAll();
                 await ProcessTableUpdate(table, "BecomeActive", true);
                     
-                GossipMyStatus(); // only now read and stored the table locally.
+                GossipMyStatus().Ignore(); // only now read and stored the table locally.
```

Comment:
Unsure of the purpose of this. Are we gossiping our status twice here, once in UpdateMyStatusGlobal and again immediately after?

Reply:
not sure why. But it was there before. If we conclude this line is useless and want to remove it, we can do it in another PR IMO. But the comment besides it indicates it might be here for a reason.